benchmarks/graph_infra/arkouda.graph

```diff
@@ -94,6 +94,12 @@ files: substring_search.dat, substring_search.dat, substring_search.dat
 graphtitle: Substring Search Performance
 ylabel: Performance (GiB/s)
 
+perfkeys: non-regex flatten with literal delimiter Average rate =, regex flatten with literal delimiter Average rate =, regex flatten with pattern delimiter Average rate =
+graphkeys: non-regex flatten with literal delimiter GiB/s, regex flatten with literal delimiter GiB/s, regex flatten with pattern delimiter GiB/s
+files: flatten.dat, flatten.dat, flatten.dat
+graphtitle: Flatten Performance
+ylabel: Performance (GiB/s)
+
```

Comment:
This is minor and only impacts what suite the generated graphs end up in, but could you move the substring/flatten graphs into `arkouda-string.graph`?